manifests/gcp_marketplace/schema.yaml

```diff
@@ -0,0 +1,15 @@
+applicationApiVersion: v1beta1
+
+properties:
+  name:
+    type: string
+    x-google-marketplace:
+      type: NAME
+  namespace:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+
```

Comment:
Include one property for each container image (besides the deployer itself) that you'll use in your app.
https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/building-deployer-helm.md#parameterize-the-images

The app might still install with `mpdev install`, but our system relies on properties with x-google-marketplace type: IMAGE for determining which images your app uses (both for publishing them, and then to properly reference those published images at deploy time).